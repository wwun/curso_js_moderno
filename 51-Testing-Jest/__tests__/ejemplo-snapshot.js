//los snapshot son datos qe se almacenan en un string para poder compararlo
//npm test t -- -u

const cliente = {
    nombre: 'William Wun',
    balance: 500,
    tipo: 'premium'
}

describe('testing al cliente', () => {
    test('es William Wun', () => {
        expect(cliente).toMatchSnapshot();  //crea una carpeta __snapshots__ con lo qe convierte jest   npm t -- -u actualiza los snapshot cuando se cambian los valores de la prueba
    });
});