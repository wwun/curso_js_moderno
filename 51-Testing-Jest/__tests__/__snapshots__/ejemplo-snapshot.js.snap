// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing al cliente es William Wun 1`] = `
{
  "balance": 500,
  "nombre": "William Wun",
  "tipo": "premium",
}
`;
